apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/kontoregisterservice/command/SendOppdaterKontoregisterCommand.java

```diff
@@ -0,0 +1,54 @@
+package no.nav.dolly.bestilling.kontoregisterservice.command;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.dolly.bestilling.kontoregisterservice.dto.OppdaterKontoRequestDto;
+import no.nav.dolly.util.WebClientFilter;
+import no.nav.testnav.libs.dto.tpsmessagingservice.v1.TpsMeldingResponseDTO;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.util.retry.Retry;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.stream.Stream;
+
+import static java.util.Objects.nonNull;
+
+@RequiredArgsConstructor
+@Slf4j
+public class SendOppdaterKontoregisterCommand implements Callable<List<TpsMeldingResponseDTO>> {
+    private static final String KONTOREGISTER_API_URL = "/kontoregister/api/kontoregister/v1/oppdater-konto";
```

Comment:
De følger ikke noen form for standard på url ...

Antar du trenger flere post-kommandoer? og da kan det være greit å ta inn en path også som del av url.

Reply:
fullt url er sånt https://sokos-kontoregister-person.dev.intern.nav.no/kontoregister/api/kontoregister/v1/oppdater-konto
de har flere "kontoregister" i path.. kanskje fordi samler 2 tjenester på same "/kontoregister/api", litt usikker.

Reply:
Ja port og host ligger allerede i webClient. og fortsettelsen burde ha vært `/api/v1/kontoregister/ ` men, det er en annen skål. 

Mitt poeng var at hvis det følger noe mer etter `/kontoregister/api/kontoregister/v1/oppdater-konto` kunne det tas inn i egen url parameter slik at command kunne gjenbrukes til norsk og utenlandsk etc.